classes/class-wc-connect-taxjar-integration.php

```diff
@@ -379,7 +379,12 @@ public function _error( $message ) {
 			}
 
 			// if on checkout page load (not ajax), don't set an error as it prevents checkout page from displaying
-			if ( ( is_cart() || ( is_checkout() && is_ajax() ) ) && ! wc_has_notice( $message, 'error' ) ) {
+			if ( (
+					( is_cart() || ( is_checkout() && is_ajax() ) ) ||
```

Comment:
Is it possible that `is_ajax()` should be `! is_ajax()`? The original comment seems to say that if the checkout page is loaded, not via ajax, it should set the error, but the code says if it `is_checkout` and it `is_ajax`, set the error. Or am I not understanding it correctly?

Reply:
yeah, from what i see, the code wants the error to be displayed if the ajax is called.